### DISCUSSION TITLE
💡 Should tokens be prefixed by alias

### DISCUSSION THREAD

**Discussion by fraincs** [gsoft-inc/sg-orbit#1111] [Ideas]
## Problem to solve

Orbit tokens are prefixed with `alias` when they refer to another token. While looking at others 99% of them don't use alias for token names. Tokens names if explicit enough shouldn't need alias in their name as `bg-accent` intent is clear enough. Alias pollutes autocompletion too 

![image](https://user-images.githubusercontent.com/361632/151196416-8fd3fd61-27d3-4274-9bb7-d548bb601e22.png)

Here's an example of other system token naming convention : 

![image](https://user-images.githubusercontent.com/361632/151197406-e52ba1cd-abf4-42cb-a0ac-70847f75ee0c.png)

## Idea

Remove alias in token names, documentation should be enough to expose tokens that should be used. If this becomes a problem a linter could be added in order to help with the usage of tokens.

Doing this would require some restructuration of token naming e.g. text color tokens don't have text-color in their name this could lead to weird token names  `default` is not a good token name `text-color-default` is much better.

**Comment by patricklafrance** [comment]
I think a linter could be added from day one. I doubt it would be a hard one to code, @alexasselin008 is a pro.

**Comment by patricklafrance** [comment]
Is it still relevant? If so, I think it should be moved to a Github Discussion until a formal proposal is available.
